src/base/configset.lua

```diff
@@ -61,22 +61,30 @@
 --    blocks with terms fully contained by this list will be considered in
 --    determining the returned value. Terms should be lower case to make
 --    the context filtering case-insensitive.
+-- @param onlylocal
+--    If true, don't fetch values from parent configs.
```

Comment:
Nitpick; It looks like this comment might have meant `origin` instead? Either that or it's mismatched? The description doesn't look like what the `origin` param appears to be.